src/workers.cc

```diff
@@ -378,14 +378,14 @@ void KafkaConsumerConsumeLoop::Execute(const ExecutionMessageBus& bus) {
       // when in consume loop mode
       // Randomise the wait time to avoid contention on different
       // slow topics
-      usleep(static_cast<int>(rand_r(&m_rand_seed) * 1000 * 1000 / RAND_MAX));
+      std::this_thread::sleep_for(std::chrono::milliseconds(static_cast<int>(rand() * 1000 / RAND_MAX)));
```

Comment:
Good catch @vsimonian.  C++ isn't my strong suit so thanks for the tip, I'll check in a change with an IFDEF momentarily.